tlds/gen.go

```diff
@@ -0,0 +1,58 @@
+//+build ignore
+
+package main
+
+import (
+	"bytes"
+	"go/format"
+	"io/ioutil"
+	"os"
+	"strings"
+
+	"github.com/alecthomas/template"
```

Comment:
Given that the templating is fairly minimal are you open to constructing the template string using the stdlib `	"text/template"` like the expiration-mailer instead of adding a new dep?